public/app/plugins/panel/geomap/components/DataHoverView.tsx

```diff
@@ -35,17 +38,29 @@ export class DataHoverView extends PureComponent<Props> {
       return null;
     }
 
+    const displayValues: Array<[string, string]> = [];
+    const visibleFields = data.fields.filter((f) => !Boolean(f.config.custom?.hideFrom?.tooltip));
+
+    if (visibleFields.length === 0) {
+      return null;
+    }
+    for (let i = 0; i < visibleFields.length; i++) {
+      displayValues.push([getFieldDisplayName(visibleFields[i], data), fmt(visibleFields[i], rowIndex!)]);
```

Comment:
this will force everything to a string *before* sort -- that is reasonable, but may have weird behavior with numbers (eg: 10, 5000, 400) 

Should we delay calling `fmt()` until react?  Then the sort will be based on the raw values, but may mix strings+numbers :thinking:

Reply:
yup, changed!